client/pages/sign-in.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import SignupForm from '../components/signup-form';
+
+class SignInPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: []
+    };
+    this.addNewUser = this.addNewUser.bind(this);
+  }
+
+  componentDidMount() {
+    fetch('/api/users')
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ users: data });
+      })
+      .catch(err => console.error(err));
+  }
+
+  addNewUser(newUser) {
+    fetch('/api/auth/sign-up', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(newUser)
+    })
+      .then(response => response.json())
+      .then(newUser => {
+        this.setState({ users: [].concat(this.state.users, newUser) });
+      })
+      .catch(err => console.error(err));
```

Comment:
Once the user is signed up, you can switch them to the sign in page (which can be blank for now)

```js
window.location.hash = 'sign-in'
```